js/livecoin.js

```diff
@@ -502,8 +502,10 @@ module.exports = class livecoin extends Exchange {
         let response = await this.privateGetExchangeClientOrders (this.extend (request, params));
         let result = [];
         let rawOrders = [];
-        if (response['data'])
+        if (response['data']) {
             rawOrders = response['data'];
+            rawOrders = rawOrders.reverse();
```

Comment:
I suggest to move this edit to line 513:

```
return this.sortBy (result, 'timestamp');
```

That should work and transpile properly. Can you edit it slightly, plz?